data/styles/clear/include/Roads.mapcss

```diff
@@ -1173,33 +1197,50 @@ line|z14-[railway=yard][tunnel?]
 
 /* 9.3 Rail bridge 14-22 ZOOM */
 
+/* TODO
+  remove following rules
+  add casing-color and z-index definitions
+  to test these new bridge styles
```

Comment:
What for are this definitions here? What do they fix?

Reply:
So evaluation time of the final width was changed to happen after all rules were processed.
This led to addition of a few "new" (and incomplete, e.g. w/o color defined) bridge rules into the compiled style - e.g. monorail lines widths were defined after the bridge section and hence they were not processed previously but are processed now.

This definition is just a plug to keep these styles "disabled" for now, because a proper addition of them is a lot of separate work.
In this PR I tried to keep real style changes to a very minimum.